sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -226,6 +232,38 @@ void _nib_nc_remove(_nib_onl_entry_t *node)
     _nib_onl_clear(node);
 }
 
+static inline void _get_l2addr_from_ipv6(uint8_t *l2addr,
+                                         const ipv6_addr_t *ipv6)
+{
+    memcpy(l2addr, &ipv6->u64[1], sizeof(uint64_t));
+    l2addr[0] ^= 0x02;
```

Comment:
It seems way more efficient to me to just toggle, than checking and then clearing / setting, so I'm not sure what you mean.